kratos/utilities/time_discretization.cpp

```diff
@@ -0,0 +1,150 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+// System includes
+#include <limits>
+
+// External includes
+
+// Project includes
+#include "includes/checks.h"
+#include "time_discretization.h"
+
+namespace Kratos {
+namespace TimeDiscretization {
+
+std::array<double, 2> BDF1::ComputeBDFCoefficients(const double DeltaTime) const
+{
+    KRATOS_ERROR_IF(DeltaTime < std::numeric_limits<double>::epsilon())
+        << "Expects DeltaTime > 0!" << std::endl;
+
+    std::array<double, 2> coefficients;
+
+    coefficients[0] =  1.0/DeltaTime;
+    coefficients[1] = -1.0/DeltaTime;
+
+    return coefficients;
+}
+
+std::array<double, 3> BDF2::ComputeBDFCoefficients(const double DeltaTime) const
```

Comment:
I am not too positive about leaving this version ... my point is that people will use it instead of the "correct" one that handles the variable dt case ... so i would suggest removing it completely

Reply:
Ok